src/pages/wishlist/Wishlist.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import axios from "axios";
+import { useEffect } from "react";
+import "./Wishlist.css";
+import { useWishlist } from "../../contexts/Wishlist-Context";
+import { useCart } from "../../contexts/Cart-Context";
+import { axiosRemoveFromWishlist } from "../../utilty/wishlist-utility";
+import { axiosAddToCart } from "../../utilty/cart-utility";
+
+export function Wishlist() {
+  const { wishlistItems, setwishlistItems } = useWishlist();
+  const { cartState, cartDispatch } = useCart();
+
+  useEffect(() => {
```

Comment:
Create a file called `server-requests` or any other name of your choice and add all your SERVER REQUESTS in that (GET, POST, etc).

Reply:
Just realized , that request wasn't needed, deleted it.